examples/nlp/language_modeling/bert_pretraining.py

```diff
@@ -404,9 +404,11 @@ def create_pipeline(data_file, batch_size, preprocessed_data=False, batches_per_
 # callback which prints training loss and perplexity once in a while
 if not args.only_mlm_loss:
     log_tensors = [train_loss, mlm_loss, nsp_loss]
+    eval_log_tensors = [eval_loss, eval_mlm_loss, eval_nsp_loss]
     print_msg = "Loss: {:.3f} MLM Loss: {:.3f} NSP Loss: {:.3f}"
 else:
     log_tensors = [train_loss]
+    eval_log_tensors = [eval_loss]
     print_msg = "Loss: {:.3f}"
 train_callback = nemo_core.SimpleLossLoggerCallback(
     tensors=log_tensors,
```

Comment:
If you want to log evaluation losses in the TB, you should add them here too.

Reply:
ok